src/common/string/compare.ts

```diff
@@ -1,13 +1,8 @@
-export const stringCompare = (a: string, b: string) => {
-  if (a < b) {
-    return -1;
-  }
-  if (a > b) {
-    return 1;
-  }
+export const stringCompare = (a: string, b: string, language: string) =>
+  a.localeCompare(b, language);
 
-  return 0;
-};
-
-export const caseInsensitiveStringCompare = (a: string, b: string) =>
-  stringCompare(a.toLowerCase(), b.toLowerCase());
+export const caseInsensitiveStringCompare = (
+  a: string,
+  b: string,
+  language: string
```

Comment:
Make language optional, when it is not provided, the browser language is used

Reply:
👍

Reply:
Done.